app/services/comments/destroy.rb

```diff
@@ -0,0 +1,37 @@
+module Comments
+  class Destroy
+
+    def self.call(article_id, comment_id)
+      new(article_id, comment_id).call
+    end
+
+    def call
+      destroy
+      errors
```

Comment:
Return `self` to return the whole `Comments::Destroy` object to be able to call `.errors` in the controller.